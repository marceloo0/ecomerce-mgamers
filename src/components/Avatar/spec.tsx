import { screen } from '@testing-library/react'
import { renderWithTheme } from '../../utils/tests/helpers'

import { Avatar } from '.'

describe('<Avatar />', () => {
  it('should render default', () => {
    const { container } = renderWithTheme(<Avatar />)

    expect(screen.getByText(/M/i)).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
