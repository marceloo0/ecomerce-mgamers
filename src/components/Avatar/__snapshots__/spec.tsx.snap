// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Avatar /> should render default 1`] = `
.c0 {
  width: 2rem;
  height: 2rem;
  border: 3px solid #7FFA45;
  border-radius: 50%;
  background-color: #201C2B;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c1 {
  text-transform: uppercase;
  color: #C7ECEC;
  font-size: 1.2rem;
  font-weight: 600;
}

@media screen and (min-width:768px) {
  .c0 {
    width: 2.2rem;
    height: 2.2rem;
  }
}

@media screen and (min-width:768px) {
  .c1 {
    font-size: 1.6rem;
  }
}

<div
  class="c0"
>
  <span
    class="c1"
  >
    m
  </span>
</div>
`;
